shenyu-admin/src/main/java/org/apache/shenyu/admin/model/entity/ScalePolicyDO.java

```diff
@@ -0,0 +1,385 @@
+     *
+     * @return the value of status
+     */
+    public Integer getStatus() {
+        return status;
+    }
+
+    /**
+     * Sets the status.
+     *
+     * @param status status
+     */
+    public void setStatus(final Integer status) {
+        this.status = status;
+    }
+
+    /**
+     * Gets the value of num.
+     *
+     * @return the value of num
+     */
+    public Integer getNum() {
+        return num;
+    }
+
+    /**
+     * Sets the num.
+     *
+     * @param num num
+     */
+    public void setNum(final Integer num) {
+        this.num = num;
+    }
+
+    /**
+     * Gets the value of beginTime.
+     *
+     * @return the value of beginTime
+     */
+    public Date getBeginTime() {
+        return beginTime;
+    }
+
+    /**
+     * Sets the beginTime.
+     *
+     * @param beginTime beginTime
+     */
+    public void setBeginTime(final Date beginTime) {
+        this.beginTime = beginTime;
+    }
+
+    /**
+     * Gets the value of endTime.
+     *
+     * @return the value of endTime
+     */
+    public Date getEndTime() {
+        return endTime;
+    }
+
+    /**
+     * Sets the endTime.
+     *
+     * @param endTime endTime
+     */
+    public void setEndTime(final Date endTime) {
+        this.endTime = endTime;
+    }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        if (!super.equals(o)) {
+            return false;
+        }
+        ScalePolicyDO that = (ScalePolicyDO) o;
+        return Objects.equals(sort, that.sort)
+                && Objects.equals(status, that.status)
+                && Objects.equals(num, that.num)
+                && Objects.equals(beginTime, that.beginTime)
+                && Objects.equals(endTime, that.endTime);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(super.hashCode(), sort, status, num, beginTime, endTime);
+    }
+
+    /**
+     * builder.
+     *
+     * @return ScalePolicyDOBuilder
+     */
+    public static ScalePolicyDO.ScalePolicyDOBuilder builder() {
+        return new ScalePolicyDO.ScalePolicyDOBuilder();
+    }
+
+    /**
+     * buildScalePolicyDO.
+     *
+     * @param scalePolicyDTO scalePolicyDTO
+     * @return ScalePolicyDO
+     */
+    public static ScalePolicyDO buildScalePolicyDO(final ScalePolicyDTO scalePolicyDTO) {
+        return Optional.ofNullable(scalePolicyDTO).map(item -> {
+            Timestamp currentTime = new Timestamp(System.currentTimeMillis());
+            try {
+                ScalePolicyDO scalePolicyDO =
+                        ScalePolicyDO.builder()
+                                .sort(item.getSort())
+                                .status(item.getStatus())
+                                .num(item.getNum())
+                                .beginTime(DateUtils.isValidDate(item.getBeginTime())
```

Comment:
Why is the begintime null?

Reply:
when use manual strategy or dynamic strategy, there is not need to fill it.